PR: https://github.com/odoo/odoo/pull/

From: 8a4635a4e209db45162dacb3f4ec174ec786789a
From: Christophe Simonis
Date: 2016-09-02 11:58:27

Structural Changes: 6
Total Changes: 6

[FIX] project{,_issue}: mark related fields as readonly

================================= pseudo patch: =================================

--- a/addons/project_issue/models/project_issue.py
+++ b/addons/project_issue/models/project_issue.py
@@ -60,9 +60,9 @@ class ProjectIssue(models.Model):
     user_email = fields.Char(related='user_id.email', string='User Email', readonly=True)
     date_action_last = fields.Datetime(string='Last Action', readonly=True)
     date_action_next = fields.Datetime(string='Next Action', readonly=True)
-    legend_blocked = fields.Char(related="stage_id.legend_blocked", string='Kanban Blocked Explanation')
-    legend_done = fields.Char(related="stage_id.legend_done", string='Kanban Valid Explanation')
-    legend_normal = fields.Char(related="stage_id.legend_normal", string='Kanban Ongoing Explanation')
+    legend_blocked = fields.Char(related="stage_id.legend_blocked", string='Kanban Blocked Explanation', readonly=True)
+    legend_done = fields.Char(related="stage_id.legend_done", string='Kanban Valid Explanation', readonly=True)
+    legend_normal = fields.Char(related="stage_id.legend_normal", string='Kanban Ongoing Explanation', readonly=True)
 
     @api.multi
     def _read_group_stage_ids(self, domain, read_group_order=None, access_rights_uid=None):
